### ISSUE TITLE
How to get the dictionary returned by the validation_epoch_end() method?

### ISSUE THREAD

**Issue by nrjvarshney** [PyTorchLightning/pytorch-lightning#2602]
#### What is your question?
How do I fetch the dictionary returned by the validation_epoch_end() method? I don't want to add this to tensorflow logs 
#### Code

   

return {"label_freq": label_freq, log: {"loss": loss}}

I want to get the value of label_freq

#### What's your environment?

 - OS: [e.g. iOS, Linux, Win]
 - Packaging [e.g. pip, conda]
 - Version [e.g. 0.5.2.1]

**Comment by rohitgr7** [comment]
Do you want this after every `validation_epoch_end` or for just the last epoch?

**Comment by nrjvarshney** [comment]
After every epoch. @rohitgr7
